### DISCUSSION TITLE
Help Needed with Site Verification on Mastodon.art instance

### DISCUSSION THREAD

**Discussion by DrunkDuckAdmin** [mastodon/mastodon#20324] [Q&A]
Hi everyone. Got here by recommendation of the mastodon.art instance admin. I'm the admin for DrunkDuck and make our new mastodon account after migrating from twitter (this is our handle @drunkduck@mastodon.art). We followed instructions and put the verification code on the landing page of our site https://www.theduckwebcomics.com/ but it's been more than 72 hours now and we have no checkmark/verification badge or however it shows up that we are the owners of the particular site link.

Can you help? We're artists rather than coders so treat us like the ignoramuses that we definitely are. Thank you in advance!

**Comment by MrGeneration** [comment]
I think you're talking about the link verification of profiles.
I did have a look at the site but couldn't find any Mastodon handles.

Any way. Modoboa expects at the following attribute on `a href` tags: `rel="me"`.
The link it expects is the link to the Mastodon profile as `https://mastodon.art/@drunkduck` to but it in your sample.

It can't verify the link on your profile if the specific site you provide does not contain said link.

**Reply by DrunkDuckAdmin** [reply]
thank you very much! I'll relay it.
